fifo

```diff
@@ -322,22 +335,22 @@ format_partitions() {
     read_input_text "Confirm format $1 partition"
     if [[ $OPTION == y ]]; then
       [[ -z $3 ]] && select_filesystem || filesystem=$3
-      mkfs.${filesystem} $1 \
-        $([[ ${filesystem} == xfs || ${filesystem} == btrfs || ${filesystem} == reiserfs ]] && echo "-f") \
-        $([[ ${filesystem} == vfat ]] && echo "-F32") \
-        $([[ $TRIM -eq 1 && ${filesystem} == ext4 ]] && echo "-E discard")
-      fsck $1
-      mkdir -p $2
-      mount -t ${filesystem} $1 $2
+      mkfs."${filesystem}" "$1" \
```

Comment:
Seems to break the step of actually formatting the chosen root partition.. 

I got error: mkfs.ext4: invalid blocks `` on device `/dev/sda6`

Reverting last 2 merges fixes it for me.

Reply:
![Screenshot 2020-06-25 at 15 25 30](https://user-images.githubusercontent.com/677051/85728528-4c82b300-b6f8-11ea-986d-67c9d817c108.png)
I have fixed this, but I will do more tests to confirm that everything is working as expected.